minutes/2019-11-12.md

```diff
@@ -0,0 +1,65 @@
+# Jakarta EE Platform Call #2
+
+General
+Date: 2019-11-12
+
+Present:
+
+* Kevin Sutter
+* Bill Shannon
+* Ivar Grimstad
+* Emily Jiang
+* Tanja Obradovic
+* Martin Stefanko
+* Ken Finnigan
+* Edwin Amoakwa
+* Carlos Andres De La Rosa
+* Steve Millidge
+* Rudy De Busscher
+* Alessio Soldano
+
+## Agenda
+
+* Organize ourselves
+
+Kevin (IBM) and Steve (Payara) volunteered to take on the Delivery Owner and Co-pilot role for Jakarta EE 9. Final confirmation pending.
+
+Keep the current meeting time: 11:00 AM EST.
+
+Minutes will be published at 
+(Jakarta EE Platform Project Web Page)https://eclipse-ee4j.github.io/jakartaee-platform/ 
+
+Source of minutes:
+https://github.com/eclipse-ee4j/jakartaee-platform/blob/gh-pages/minutes
+
+
+* [KWS] Big Bang vs Incremental.  Need to decide.  Several projects are stomping at the bit to move on the package rename exercise.
+
+**DECISION**:  Big Bang is the stated direction.
+
+* [KWS] Java SE 8 or Java SE 11 as the minimum.  Another key decision to allow for continued development efforts.
+
+* [IG] “Jakartify” the specification documents.
+Status: https://github.com/eclipse-ee4j/jakartaee-platform/projects/2
+
+* [IG] Decide which specifications to prune/deprecate.
+Publish specific proposal(s) to Platform dev mailing list.  Steve volunteered to organize the material and post for a “vote” by Platform Committers.
+
+* [TO] Tanja to find out a list of committers on the Jakarta EE Platform Project and invite them to sign up to the platform project mailing list - look here
```

Comment:
I think the existing platform project committers need to know the list of platform project committers who
are not (or will not) sign up for the mailing list.  Such committers probably should not be committers.

Reply:
I stated that we require all committers to be subscribed...